msmbuilder/cluster/agglomerative.py

```diff
@@ -142,6 +142,8 @@ class _LandmarkAgglomerative(ClusterMixin, TransformerMixin):
     ----------
     landmark_labels_
     landmarks_
+    cluster_centers_ : mean of all points in each cluster along every dimension,
```

Comment:
Ah okay, I misunderstood. Maybe just leave this as the default sklearn description for `cluster_centers_` (with a note about rmsd)? Something like:

>  Coordinates of cluster centers (unless RMSD is the metric)

Could you also follow the sklearn docstring template for attributes?

```
    Attributes
    ----------
    attribute_ : var_type, [dim1, dim2]
        Description here.
```